frontend/components/CookieBar.test.js

```diff
@@ -0,0 +1,30 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import { getCookieConsentValue } from "react-cookie-consent";
+import jestConfig from "../jest.config";
+import CookieBar from "./CookieBar";
+
+describe("Cookiebar", () => {
+  const onAcceptMock = jest.fn();
+
+  beforeEach(() => {
+    render(<CookieBar onAccept={onAcceptMock} />);
+  });
+
+  it("is rendered", () => {
+    expect(screen.queryByText("Accept cookies")).toBeDefined();
+  });
+  it("disappears when you click a button", async () => {
```

Comment:
This is pretty nitpicky, but I'm unsure why this function is `async` when nothing inside it uses `await`. I suspect this may be inspired by some of my tests for `SubscriptionForm`?

I got a bit tripped up whether I needed to `await` when using `fireEvent`, but I'm not convinced it's necessary. If the function doesn't use `await` anywhere, I'm pretty sure you can safely remove the `async`.

This applies to tests in a number of files.

Reply:
I've removed the async in the cookiebar and you're right, it seems to work fine. Strange that you have to add act wrappers to all the fireevents in the scenario tests.